zerver/webhooks/newrelic/doc.md

```diff
@@ -20,6 +20,6 @@ New Relic can send messages to a Zulip stream for incidents.
 
 {!congrats.md!}
 
-![](/static/images/integrations/newrelic/opened.png)
-![](/static/images/integrations/newrelic/ack.png)
-![](/static/images/integrations/newrelic/closed.png)
+![](/static/images/integrations/newrelic/001.png)
+![](/static/images/integrations/newrelic/002.png)
+![](/static/images/integrations/newrelic/003.png)
```

Comment:
I don't understand this commit; I think you might need to actually generate the screenshots (and delete the old opened/ack/closed ones) as well for this to work; otherwise you just have broken links.

Reply:
Sure, I will run the tool to get this work.